src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml

```diff
@@ -21,5 +21,8 @@
     <type fullname="System.StartupHookProvider" feature="System.StartupHookProvider.IsSupported" featurevalue="false">
       <method signature="System.Boolean get_IsSupported()" body="stub" value="false" />
     </type>
+    <type fullname="System.Threading.ThreadPool" feature="System.Threading.ThreadPool.EnableDispatchAutoreleasePool" featurevalue="false">
+      <method signature="EnableDispatchAutoreleasePool" value="false" initialize="false" />
```

Comment:
Does this work? Other similar methods in this file use a more verbose `System.Boolean get_...` for the signature.

Reply:
The other methods above are properties. This is a field, so that's why this format is used.

Reply:
But the xml tags says "method". Also, I got confused again by it being a field and not having `s_` prefix.

Reply:
`s_asyncDebuggingEnabled` is outlier (the debugger has dependency on the exact name, so it is not easy to mess with it). We use properties everywhere else - it looks better with properties.

Reply:
I'll change it to be a property so it will only be read when on a platform that reads the switch.